app.js

```diff
@@ -26,8 +25,10 @@ openevent.config(['$urlRouterProvider', '$httpProvider', function($urlRouterProv
     //delete $httpProvider.defaults.headers.common['X-Requested-With'];
     $urlRouterProvider.otherwise('sessions');
     }]);
+
+var totalDays;
```

Comment:
Because I am using it in app.js and also again in SessionController.js }
What is the best way to have a global js variable ?